{
	"deprecated": false,
	"cpeName": "cpe:2.3:a:python:setuptools:58.0.0:*:*:*:*:*:*:*",
	"cpeNameId": "B182D5C8-48B1-4455-AC20-EE7000534F7F",
	"created": "2022-12-29T14:39:02.883",
	"lastModified": "2022-12-29T14:43:10.977",
	"titles": [
		{
			"title": "Python setuptools 58.0.0",
			"lang": "en"
		}
	],
	"refs": [
		{
			"ref": "https://pypi.org/project/setuptools/#history",
			"type": "Version"
		},
		{
			"ref": "https://pyup.io/posts/pyup-discovers-redos-vulnerabilities-in-top-python-packages/",
			"type": "Advisory"
		}
	]
}
